dali/sasha/saserver.cpp

```diff
@@ -365,9 +460,6 @@ int main(int argc, const char* argv[])
                     ISashaServer &server=servers.item(i1);
                     server.start();
                 }
-
-                SashaServerStatus = new CSDSServerStatus("SashaServer");
```

Comment:
I removed because unused afaik - the only possible use it could serve, would be for esp service discovery in bare-metal, but I couldn't see that it was used.
@wangkx - please verify

Reply:
I just searched the code in esp. I haven't found any place where it is used.

Reply:
ok thanks.